packages/amesos2/src/Amesos2_Superlu_def.hpp

```diff
@@ -752,6 +752,7 @@ Superlu<Matrix,Vector>::getValidParameters_impl() const
 
     // ILU parameters
 
+#if 0
```

Comment:
This "if 0" worries me. We have to support older SuperLU and I am not sure this has been removed everywhere.